### PR TITLE
Question component refactor

### PR DESCRIPTION
**Pull request by desplodur**
Please take a look at my latest PR

### CODE REVIEW

File: src/components/QuestionCard/index.js [hunk 1]
```diff
@@ -1,188 +1,35 @@
...
-							!question.answered
-								? require('../../utils/icons/statusOpen.svg').default
-								: require('../../utils/icons/statusClosed.svg').default
-						}
-						alt={!question.answered ? 'Open' : 'Closed'}
-					/>
-				)}
+				<h4>{question.questionText}</h4>
+				<img
+					id="statusIndicator"
```
Comment (JessicaLoers) [inline src/components/QuestionCard/index.js:16]: What are you doing with this, a kind of weird id? 🤔

File: src/components/QuestionDetails/index.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+				>
+					<svg width="2em" height="2em" viewBox="0 0 24 24">
+						<path
+							fill="currentColor"
+							d="M2,12A10,10 0 0,1 12,2A10,10 0 0,1 22,12A10,10 0 0,1 12,22A10,10 0 0,1 2,12M18,11H10L13.5,7.5L12.08,6.08L6.16,12L12.08,17.92L13.5,16.5L10,13H18V11Z"
+						/>
+					</svg>
+				</Button>
+				<StyledImage
+					id="statusIndicator"
```
Comment (JessicaLoers) [inline src/components/QuestionDetails/index.js:71]: What are you doing with this, a kind of weird id? 🤔

**Review by JessicaLoers — APPROVED** [review]
**Well done** ✌️
I have just a question about the `<StyledImage> id`

File: src/components/QuestionDetails/index.js (hunk 2 above)
Comment (desplodur) [reply src/components/QuestionDetails/index.js:71]: i`ve replaced it with an styled component :)

**Pull request merged**